src/app/shared/store/registration.state.ts

```diff
@@ -131,7 +131,7 @@ export class RegistrationState {
   }
 
   @Action(Login)
-  login({}: StateContext<RegistrationStateModel>, { payload }: Login): void {
+  login(_: StateContext<RegistrationStateModel>, { payload }: Login): void {
```

Comment:
Also need to confirm the approach. The underscore is used to avoid SonarLint unexpected object pattern code smell and marking the argument as ignored; note that injecting StateContext here is necessary for method to perform.

Reply:
The underscore at the beginning of the unsed parameter is okay, but it's better to add more sense, maybe do something like `_context` or `_ctx`.